SampleApp/AutocompleteTableVC.swift

```diff
@@ -27,21 +27,19 @@ class AutocompleteTableVC: UITableViewController, UISearchResultsUpdating, UISea
     self.definesPresentationContext = true
     searchController.searchBar.sizeToFit()
     manager.delegate = self
-    PeliasSearchManager.sharedInstance.urlQueryItems = [NSURLQueryItem(name: "apiKey", value: "mapzen-2qQR7SX")]
+    PeliasSearchManager.sharedInstance.urlQueryItems = [NSURLQueryItem(name: "api_key", value: "mapzen-2qQR7SX")]
```

Comment:
We should create an issue to remove hard-coded API keys prior to launch.

The sample app in the Android SDK has developers provide their own Mapzen API key via an [xml resource file](https://github.com/mapzen/android/blob/master/samples/mapzen-android-sdk-sample/src/main/res/values/mapzen.xml) or [Gradle properties](https://github.com/mapzen/android/blob/master/samples/mapzen-android-sdk-sample/build.gradle#L4).

What we the best approach on iOS? A `.plist` file?